src/geode/basic/string.cpp

```diff
@@ -35,37 +35,40 @@ namespace geode
 
     index_t string_to_index( absl::string_view string )
     {
-        index_t value;
-        const auto ok = absl::SimpleAtoi( string, &value );
-        OPENGEODE_EXCEPTION( ok, "[string_to_index] Cannot convert string \"",
-            string, "\" to index" );
+        index_t value{ 0 };
+        const auto status = absl::SimpleAtoi( string, &value );
+        OPENGEODE_EXCEPTION( status,
+            "[string_to_index] Cannot convert string \"", string,
+            "\" to index" );
         return value;
     }
 
     int string_to_int( absl::string_view string )
     {
-        int value;
-        const auto ok = absl::SimpleAtoi( string, &value );
-        OPENGEODE_EXCEPTION( ok, "[string_to_int] Cannot convert string \"",
+        int value{ 0 };
```

Comment:
warning: variable 'value' of type 'int' can be declared 'const' [misc-const-correctness]

```suggestion
        int const value{ 0 };
```